packer/scripts/create-kubedns.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -o errexit -o xtrace
+
+echo "Waiting for kube-apiserver to be active..."
+while ! systemctl is-active kube-apiserver.service 2>/dev/null >/dev/null; do
+  sleep 10
+done
+echo "Waiting for kubectl to respond..."
+while ! kubectl get pods --all-namespaces; do
+  sleep 2
+done
+# Due to timing, kube-system may not exist immediately
+while ! kubectl get ns kube-system >& /dev/null; do
+  sleep .1
+done
+perl -p -i -e '
+  s@clusterIP:.*@clusterIP: 10.254.0.254@ ;
+  s@170Mi@256Mi@ ;
+  s@70Mi@128Mi@ ;
+' /etc/kubernetes/addons/kubedns.yml
+kubectl create --namespace kube-system --filename /etc/kubernetes/addons/kubedns.yml
```

Comment:
I'm not exactly sure how the IPs in the SubjectAltName are used. In any event, the apiserver cert is the only one that got this before (when 192.168.77.77 was assumed). This branch hasn't touched [this](https://github.com/SUSE/scf/blob/harts-vagrant-use-public-network-rebased/packer/scripts/create-k8s-certs.sh#L7) With single-node deployments (such as the vagrant deployment), the other kube services are talking to the apiserver via 127.0.0.1, so maybe this is why things work regardless of the IP used to access the host from outside... I can verify that things all work the same with a `10.` IP.

My understanding is that the vagrant image/deployment is currently not set up in such a way that authenticated client access is even possible. The kube api *is* exposed at https://$IP:6443, and I'm able to communicate with it via kubectl. It's worth noting that cf-dev.io isn't in the apiserver cert's domain list right now either, but from the client machine, if I set `kubernetes` to the right IP in my /etc/hosts, and use `https://kubernetes:6443`, I don't get any errors mentioning the certificate domain. Even still, kubectl prompts for user/password credentials, and I don't believe any user has been configured.

So right now, I think this is out of scope for this PR.